spec/features/authentication/set_up_enhanced_two_factor_authentication_spec.rb

```diff
@@ -62,17 +94,55 @@
       expect(page).to have_css("#resend-email")
     end
 
-    it "asks for the email 2FA method when user logs in again" do
-      click_on "Sign out"
+    it "sends an email" do
+      expect(notification_instance).to have_received(:send_email)
+    end
 
-      sign_in_user(user, pass_through_two_factor: false)
-      visit root_path
+    context "the user resends the email" do
+      before :each do
+        click_link "resend-email"
+      end
+      it "explains the email can be sent again" do
+        expect(page).to have_content("Emails sometimes take a few minutes to arrive.")
+        expect(page).to have_content("If you do not receive the email, we can send you a new one.")
+      end
+      it "does not send further emails" do
+        expect(notification_instance).to have_received(:send_email).once
+      end
+      context "the user confirms" do
+        before :each do
+          click_on "Resend email"
+        end
+        it "sends another email" do
```

Comment:
let's space all the `it` block in newlines

Reply:
👍